__tests__/app.test.js

```diff
@@ -39,4 +39,39 @@ describe("/api", () => {
         expect(res.body.endpoints).toEqual(endpoints);
       });
   });
-});
\ No newline at end of file
+});
+
+describe("/api/articles/:article_id", () => {
+  test("GET - status 200 - Returns a valid article", () => {
+    return request(app)
+    .get("/api/articles/2")
+    .expect(200)
+    .then((res) => {
+      expect(res.body.article.article_id).toBe(2)
+      expect(res.body.article).toHaveProperty('author')
+      expect(res.body.article).toHaveProperty('title')
+      expect(res.body.article).toHaveProperty('topic')
+      expect(res.body.article).toHaveProperty('author')
+      expect(res.body.article).toHaveProperty('body')
+      expect(res.body.article).toHaveProperty('created_at')
+      expect(res.body.article).toHaveProperty('votes')
+      expect(res.body.article).toHaveProperty('article_img_url')
+    })
```

Comment:
Rather than using toHaveProperty here, could we change to a more flexible matcher like expect.objectContaining or .toMatchObject please? The reason is because this endpoint is going to change later in the project. 🙂